var/spack/repos/builtin/packages/gnina/package.py

```diff
@@ -68,13 +68,31 @@ class Gnina(CMakePackage, CudaPackage):
     depends_on("py-numpy", type="build")
     depends_on("py-pytest", type="build")
 
-    depends_on("cuda@11")
+    depends_on("cuda@11", when="@:1.1")
+    depends_on("cuda@12", when="@1.3:")
     depends_on("cudnn", when="+cudnn")
 
+    depends_on("cmake@3.27:", when="@1.3:")  # CMake policy CMP0146 introduced in 3.27
+    depends_on("jsoncpp", when="@1.3")
+    depends_on("py-torch", when="@1.3")
```

Comment:
```suggestion
    depends_on("jsoncpp", when="@1.3:")
    depends_on("py-torch", when="@1.3:")
```
? Or do you expect them to be dropped again in the next release?

Reply:
Good catch, thanks!